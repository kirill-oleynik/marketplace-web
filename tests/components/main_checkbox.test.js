const React = require('react');
const renderer = require('react-test-renderer');
const MainCheckbox = require('../../app/components/main_checkbox').default;

describe('#render', () => {
  test('it renders correctly', () => {
    const tree = renderer.create(
      <MainCheckbox
        name="test"
        text="Test"
        value={true}
      />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
