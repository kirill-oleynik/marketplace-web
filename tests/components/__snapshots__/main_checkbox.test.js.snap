// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<div>
  <label
    className="main-checkbox"
    htmlFor="test"
  >
    <input
      checked={true}
      className="main-checkbox__input"
      name="test"
      type="checkbox"
    />
    <span
      className="main-checkbox__icon icon"
      onClick={[Function]}
    />
    <span
      className="main-checkbox__text"
    >
      Test
    </span>
  </label>
  
</div>
`;
